pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/InputManager.java

```diff
@@ -259,14 +264,6 @@ private void reorderDimsAndBuildMap()
     LOGGER.debug("_dimNames{}", _dimNames);
     LOGGER.debug("_metricNames{}", _metricNames);
     LOGGER.debug("_dateTimeNames{}", _dateTimeNames);
-    _metricNames.forEach(name -> {
```

Comment:
Removing this because the applicable list has all columns ?

Reply:
yes